### ISSUE TITLE
npm audit fails on bbc-a11y because of jquery vulnerability 

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#1625]
**Describe the bug**
A vulnerability in `bbc-a11y` has broken our deployment lifecycle due to us needing `audit-ci -low` to pass before we can deploy. 

![image](https://user-images.githubusercontent.com/7791726/56591370-8631c880-65e0-11e9-8a64-4eb90e0d64d6.png)

See https://github.com/bbc/bbc-a11y/issues/289

**Solution**
Once https://github.com/bbc/bbc-a11y/issues/289 has been fixed and published we need to pull in the new version of bbc-a11y. 

**Testing**

Dev insight: we should be able to run `npm audit` without any failures. 

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)

**Comment by sareh** [comment]
After the `bbc-a11y` fix has been published, we should pull in that change and also remove the `--whitelist jquery` added in this PR: https://github.com/bbc/simorgh/pull/1632
